### DISCUSSION TITLE
Missing lines after php artisan lang:add sk

### DISCUSSION THREAD

**Discussion by misog** [Laravel-Lang/common#66] [Q&A]
Hi, when I run `php artisan lang:add sk` then it copies sk.json to my project. But the problem is, there are missing json keys. Certainly this file is not copied to the project: https://github.com/Laravel-Lang/lang/blob/main/locales/sk/json.json

Is it a bug or it filters out unused keys?

**Comment by andrey-helldar** [comment]
The installer does not copy files to the project. It does it wisely: it looks at what dependencies are installed in your application and installs only those keys that [match them](https://github.com/Laravel-Lang/lang/tree/main/src/Plugins).

For example:

```
namespace LaravelLang\Lang\Plugins\Jetstream;

class V3 extends Plugin
{
    protected ?string $vendor = 'laravel/jetstream';

    protected string $version = '^3.0';
}
```

Simply put, each package that connects to the installer has its own key binding to a specific product or does not have one.

The installer looks at these settings and based on them decides whether to add these translations or not.

Do I understand correctly that you need the ability to install all keys without filtering installed packages?
